test-core-docker.ini

```diff
@@ -0,0 +1,16 @@
+#
+# ckan - Pylons testing environment configuration
+#
+
+[server:main]
+use = config:test-core.ini
+
+[app:main]
+use = config:test-core.ini
```

Comment:
the test.ini has `faster_db_test_hacks = True` could that be the issue?

Reply:
It still fails even if I set `faster_db_test_hacks = False`.